src/pybind/mgr/dashboard/frontend/package.json

```diff
@@ -104,6 +104,7 @@
     "rxjs": "6.6.3",
     "simplebar-angular": "2.3.6",
     "swagger-ui": "4.12.0",
+    "table": "6.8.0",
```

Comment:
this is more like a `devDependency` right

Reply:
Ah!  yes